cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -19,11 +19,7 @@
 */
 
 body {
-    .webfont-regular();
-
-    .respond-to-max( @bp-sm-max, {
-        overflow-x: hidden;
-    } );
+    .webfont-regular(); // Moved to CF
```

Comment:
It does. Maybe something like "Moved to CF, delete freely when migrating to CF4", then?